bindings/python/iota_sdk/types/output.py

```diff
@@ -325,3 +331,34 @@ def deserialize_outputs(dicts: List[Dict[str, Any]]) -> List[Output]:
     * `dicts`: A list of dictionaries that are expected to have a key called 'type' which specifies the type of the returned value.
     """
     return list(map(deserialize_output, dicts))
+
+
+@json
+@dataclass
+class OutputIdProof:
+    """The proof of the output identifier.
+
+    Attributes:
+        slot: The slot index of the output.
+        output_index: The index of the output within the corresponding transaction.
+        transaction_commitment: The commitment of the transaction that created the output. Hex-encoded with 0x prefix.
+        output_commitment_proof: The proof of the output commitment. Hex-encoded with 0x prefix.
+    """
+    slot: SlotIndex
+    output_index: int
+    transaction_commitment: str
+    # TODO: do we need the specific types that make up the proof?
```

Comment:
🤔 who would decide this?

Reply:
After giving it some thought, now I think this has to just be a str actually containing the proof as JSON, something a user can give someone together with an output id, who then can use the sdk or a node api directly to validate it.

Reply:
I think I was wrong ... and now following the TIP specification strictly by creating those types in Python.